WinUIGallery/DataModel/ControlInfoData.json

```diff
@@ -1918,9 +1917,33 @@
             }
           ],
           "RelatedControls": [
-            "SemanticZoom",
-            "ListView",
-            "GridView"
+            "SelectorBar"
+          ]
+        },
+        {
+          "UniqueId": "SelectorBar",
+          "Title": "SelectorBar",
+          "ApiNamespace": "Microsoft.UI.Xaml.Controls",
+          "Subtitle": "Presents information from different sources in a tabbed view.",
```

Comment:
"Presents information from different sources in a tabbed view."
"SelectorBar is used to modify the content shown by allowing users to select and switch between different sets of data."

These two points are a bit misleading, I think.  The SelectorBar just lets you pick 1 option out of N, in my opinion.

"The SelectorBar is used to present a set of options. The user can pick one of them."   Something like this more general description?

Reply:
"The SelectorBar is used to present a set of options. The user can pick one of them." Sounds like a RadioButton

Maybe something like this:
Subtitle: "Presents information from a small set of different sources. The user can pick one of them."
Description: "SelectorBar is used to modify the content shown by allowing users to select and switch between a small, finite set of data."